src/python/zquantum/optimizers/cma_es_optimizer_test.py

```diff
@@ -14,7 +14,7 @@ def test_incorrect_initialization(self):
         self.assertRaises(TypeError, lambda: CMAESOptimizer())
 
     def test_cmaes_specific_fields(self):
-        cost_function = MockCostFunction()
+        cost_function = MockCostFunction().evaluate
```

Comment:
Aren't we getting rid of that? Or not yet?

Reply:
That's the next (and the last step), but for now `CostFunction` interface is still present in core, along with the `MockCostFunction`.

Reply:
Ok 👍